assets/marketplace/cloudformation/files/install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -x
```

Comment:
Debugging.

Reply:
The debugging is useful for extra output if a build fails so I'll leave it in place.